lib/block-supports/behaviors.php

```diff
@@ -99,24 +107,56 @@ function gutenberg_render_behaviors_support_lightbox( $block_content, $block ) {
 	$w->set_attribute( 'data-wp-interactive', true );
 	$w->set_attribute(
 		'data-wp-context',
-		sprintf( '{ "core":{ "image": { "initialized": false, "imageSrc": "%s", "lightboxEnabled": false, "lightboxAnimation": "%s", "hideAnimationEnabled": false } } }', $img_src, $lightbox_animation )
+		sprintf(
+			'{ "core":
+				{ "image":
+					{ 	"initialized": false,
+						"lightboxEnabled": false,
+						"hideAnimationEnabled": false,
+						"preloadInitialized": false,
+						"lightboxAnimation": "%s",
+						"imageSrc": "%s",
+						"imageSrcSet": "%s",
+						"targetWidth": "%s",
+						"targetHeight": "%s"
+					}
+				}
+			}',
+			$lightbox_animation,
+			$img_src,
+			$img_srcset ? $img_srcset : '',
```

Comment:
This line causes a warning if we use an image from URL:
![Screenshot 2023-06-23 at 12 05 37](https://github.com/WordPress/gutenberg/assets/37012961/508fe706-5cac-48d9-8097-f607d31c4514)